### PR TITLE
Upgrade the docs for Knative Operators

### PR DESCRIPTION
**Pull request by houshengbo**
## Proposed Changes 

- Upgrade the instruction on upgrading the v1alpha1 custom resources to v1beta1.

### CODE REVIEW

File: docs/install/operator/knative-with-operators.md [hunk 1]
```diff
@@ -48,6 +48,15 @@ To track the log of the Operator, run the command:
 kubectl logs -f deploy/knative-operator
 ```
 
+### Upgrade the existing custom resources
+
+If you upgrade from Knative Operator 1.2 or prior versions to the current version, run the following command to upgrade
+the existing custom resources to v1beta1 after the installation of Knative Operator:
```
[the remark this answers is not in the document]
Comment (houshengbo) [reply docs/install/operator/knative-with-operators.md:54]: cool. thx.

**Comment by dprotaso** [comment]
Does this need to be cherry picked to 1.3 branch?

File: docs/install/operator/configuring-serving-cr.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 # Configuring the Knative Serving Operator custom resource
 
-The Knative Serving Operator can be configured with the following options:
+Knative Serving can be configured with the following options:
```
Comment (snneji) [inline docs/install/operator/configuring-serving-cr.md:3]:
```suggestion
You can configure Knative Serving with the following options:
```

File: docs/install/operator/configuring-serving-cr.md [hunk 3]
```diff
@@ -36,6 +37,82 @@ enables upgrading or downgrading the Knative Serving version, without needing to
...
+There are two modes available for you to install customized manifests: _overwrite mode_ and _append mode_. With
+overwrite mode, you must define all manifests needed for Knative Serving to install because the Operator will no
+longer install any default manifests. With append mode, you only need to define your customized manifests. The
+customized manifests are installed after default manifests are applied.
+
+**Overwrite mode:**
+
+You can use overwrite mode when you want to customize all Knative Serving manifests.
+
+For example, if you want to install Knative Serving and istio ingress and you want customize both components, you can
```
Comment (snneji) [inline docs/install/operator/configuring-serving-cr.md:54]:
```suggestion
For example, if you want to install Knative Serving and Istio ingress and you want customize both components, you can
```

**Comment by csantanapr** [comment]
bump @houshengbo

**Comment by snneji** [comment]
I think since these are mostly style, formatting, and phasing fixes, we can merge this PR so that it makes it into the release. I'll make a cleanup PR to do the fixes suggested in the comments. (Clean up issue here: #4921)

**Review by snneji — APPROVED** [review]

**Pull request merged**